src/components/ResponsiveAppBar.jsx

```diff
@@ -176,23 +177,31 @@ const ResponsiveAppBar = () => {
                     sx={{ my: 2, mr:1, display: "block", textAlign: 'center'}}
                     component={Link}
                     to={`/${page.url}`}
+                    className={page.url}
                   >
                     {page.url === 'settings' ? <SettingsIcon /> : t(page.translationKey)}
                   </Button>
                 ) : null
               )}
-
           </Box>
-
-            <Button
+          <Box
+            sx={{
+              display: { xs: "none", md: "flex" },
```

Comment:
The App bar in mobile and desktop are completely different

The desktop app bar starts with 
```jsx
{/* Desktop content */}
          <Box
            sx={{
              display: { xs: "none", md: "flex" },
```

The mobile app bar starts with 
```jsx
 {/* Mobile content */}
          <Box sx={{ flexGrow: 1, display: { xs: "flex", md: "none" } }}>
```

So those modification will have no impact because on `xs` the parent is not displayed
```suggestion
              display: "flex" ,
```